modules/FormModel.js

```diff
@@ -322,40 +495,111 @@
+      var eventName = field ? ('change:' + field) : 'change';
+      var callback = field ? _.bind(this.__updateFormField, {formModel: this, field: field}) : this.__updateFormModel;
+      this.listenTo(model, eventName, callback);
       this.__currentUpdateEvents.push({model: model, eventName: eventName});
     },
 
     /**
-     * Sets up a listener on one of the fields that is needed to update a computed value
-     * @param computedConfig {Object} the Computed Config that will be updated when changes occur
+     * Sets up a listener on one (or all) of the fields that is needed to update a computed value
      * @param model {Backbone.Model} the object model from which this form model will start listen to changes
-     * @param field {String} the field name that it will start listening to.
-     * @method listenToComputedValuesDependency
+     * @param [field] {String} the field name that it will start listening to. If no field is given, it will listen to the general 'change' event.
+     * @param computedAlias {String} the name/alias of the computed mapping being used.
+     * @method __listenToComputedValuesDependency
+     * @private
      */
-    listenToComputedValuesDependency: function(computedConfig, model, field) {
-      var eventName = 'change:' + field;
-      this.listenTo(model, 'change:' + field, _.bind(this.__invokeComputedPull,
-          {formModel: this, models: computedConfig.models, pull: computedConfig.pull}));
+    __listenToComputedValuesDependency: function(model, field, computedAlias) {
+      var eventName = field ? ('change:' + field) : 'change';
+      this.listenTo(model, eventName, _.bind(this.__invokeComputedPull,
+          {formModel: this, alias: computedAlias}));
       this.__currentUpdateEvents.push({model: model, eventName: eventName});
     },
 
-    /************** Private methods **************/
+    /**
+     * Returns the models that a currently being tracked that are part of a computed mapping
+     * If there is a missing model (a model alias is referenced but no model instance is bound to that alias), then it will return undefined.
+     * @method __getComputedModels
+     * @param computedAlias {String} the name/alias of the computed mapping
+     * @return {Object} a map from model name/alias to model instance. If there is a missing model (an model alias is referenced but no model
+     *   instance is bound to that alias), then it will return undefined.
+     * @private
+     */
+    __getComputedModels: function(computedAlias) {
+      var hasAllModels = !_.isUndefined(this.getMapping(computedAlias)),
+        models = {};
+      _.each(this.__getModelAliases(computedAlias), function(modelAlias) {
+        var model = this.getTrackedModel(modelAlias);
+        if (model) {
+          models[modelAlias] = model;
+        } else {
+          hasAllModels = false;
+        }
+      }, this);
+      return hasAllModels ? models : undefined;
+    },
+
```

Comment:
I was thinking something that makes it more obvious what models aren't defined.  But I realize now I'm thinking about super complicated cases where there are more than a handful of models, which is rarely (if ever) the case.